third_party/chromium/media/base/pipeline_status.h

```diff
@@ -68,6 +68,7 @@ enum PipelineStatus {
   PLAYBACK_CAPABILITY_CHANGED = 24,
 
   PIPELINE_STATUS_MAX = PLAYBACK_CAPABILITY_CHANGED,
+  kMaxValue = PLAYBACK_CAPABILITY_CHANGED,
```

Comment:
You can avoid adding a new value to this if you'd like. Just use the enumeration call that takes a max value. Something like:

```
    base::UmaHistogramEnumeration("Cobalt.Media.PipelineStatus.AudioOnly", uma_info_.last_pipeline_status, PipelineStatus::PIPELINE_STATUS_MAX);
```

Your call though.

Reply:
done.